ODLA/platforms/dnnl/odla_dnnl.cc

```diff
@@ -1348,11 +1348,28 @@ odla_value odla_Resize(odla_value input, odla_interpolation_mode interpolation,
   auto input_md = input->mem.get_desc();
   auto dt = input->mem.get_desc().data_type();
 
-  auto ret_md = dnnl::memory::desc(getDims(output_dims), dt,
-                                   dnnl::memory::format_tag::nchw);
+  auto format_tag = dnnl::memory::format_tag::nchw;
+
+  float scale_h;
+  float scale_w;
+  std::vector<float> scales = {1.0f, 1.0f, 1.0f, 1.0f};
```

Comment:
here it can check each bit of axes_mask. if I-th bit is 0, scales[I] = 1
we don't need to worry about nhwc/nchw.